lib/msal-core/package.json

```diff
@@ -45,9 +45,11 @@
     "@types/mocha": "^5.2.6",
     "@types/node": "^11.11.5",
     "@types/sinon": "^7.0.11",
+    "@types/js-base64": "^2.3.1",
     "ajv": "6.6.1",
     "awesome-typescript-loader": "^3.2.3",
     "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
```

Comment:
This should not be added right?

Reply:
This was stated as a requirement in their release page: "Note js-base64 itself is stand-alone so its package.json has no dependencies.  However, it is also tested on ES6 environment so "babel-preset-env": "^1.7.0" is on devDependencies."

Reply:
hmmm ... 
I think what it means is that they need this to be able to run their tests, not that we need it ?
Did our tests break without it ?

Reply:
We are not testing this in E2E today, we are manually verifying. This will be in automation later. I included it for folks using ES6 to not have this issue. I am okay either way.

Reply:
I don’t believe you get anything for free including this. If it does solve a problem, it means Babel transpilation would need to be used somewhere. I think they Babel transpile their tests which is why they need it, I’ll look